src/test/java/hooks/TestHooks.java

```diff
@@ -2,24 +2,19 @@
 
 import cucumber.api.java.After;
 import cucumber.api.java.Before;
-import dataProviders.ConfigFileReader;
 import managers.WebDriverManager;
 import stepDefinition.AbstractStepDef;
 
 public class TestHooks extends AbstractStepDef {
-
-    private ConfigFileReader fileReader = new ConfigFileReader();
-
     private WebDriverManager webDriverManager = new WebDriverManager();
 
     @Before
     public void launchChromeDriver() {
-        webDriver = webDriverManager.getDriver();
+        webDriver = this.webDriverManager.getDriver();
     }
 
     @After
     public void afterScenario() {
-        webDriverManager.closeDriver();
+        this.webDriverManager.closeDriver();
```

Comment:
why should you use the keyword "this" in this method?

Reply:
Done